pkg/standalone/standalone.go

```diff
@@ -107,8 +118,13 @@ func isBinaryInstallationRequired(binaryFilePrefix, installDir string) (bool, er
 	return true, nil
 }
 
+func init() {
```

Comment:
why is this function needed ? why cannot it be a function that is called by Init() ?? try to avoid using init() as much as possible.